usl_models/usl_models/flood_ml/model_params.py

```diff
@@ -23,5 +24,17 @@ class FloodModelParams:
     # Keras parameters.
     # Keras models support both Optimizer objects and (valid) string names.
     # It is the user's responsibility to pass in a valid optimizer value.
-    optimizer: tf.keras.Optimizer | str = "adam"
-    epochs: int = 10
+    optimizer: Any = dataclasses.field(
+        default_factory=lambda: tf.keras.optimizers.Adam(learning_rate=1e-3)
+    )
+
+
+# Used for testing.
+def test_model_params() -> FloodModelParams:
+    return FloodModelParams(
+        batch_size=4,
+        m_rainfall=3,
+        n_flood_maps=3,
+        lstm_units=32,
+        lstm_kernel_size=3,
+    )
```

Comment:
Walt had initially asked for this to be removed from this file and into the `tests/` directory. Is there a reason you moved it back here?

Reply:
This was a merge snafu. Reverted.